env/dev/api-usage-plans.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_api_gateway_usage_plan" "main" {
+  name        = local.namespace
+  description = "default usage plan for ${local.namespace}"
+
+  api_stages {
+    api_id = aws_api_gateway_rest_api.main.id
+    stage  = aws_api_gateway_deployment.main.stage_name
+  }
+
+  quota_settings {
```

Comment:
I know that this usage plan is not being used by any api keys or authentication methods in the moment.

But I think the quota and throttle setting could be parametrized. Because with the current template this plan would be limited to 20 request per week. And 5 requests per seconds.

Reply:
Great point!